src/client/java/net/neoforged/neoforge/client/model/quad/BakedNormals.java

```diff
@@ -5,45 +5,142 @@
 
 package net.neoforged.neoforge.client.model.quad;
 
+import net.minecraft.client.renderer.block.model.BakedQuad;
+import org.joml.Vector3f;
+import org.joml.Vector3fc;
+import org.jspecify.annotations.Nullable;
+
 /**
  * Holds pre-computed normals, either for the entire quad or for each vertex.
  * <p>
  * The normal values are quantized to an integer between -127 and 127, truncated to a byte
  * and packed into an int, leaving the MSB unused.
+ * <p>
+ * Normals can also be unspecified. In that case, rendering will use the normal implied by {@link BakedQuad#direction()},
+ * while a face normal will be computed for the purposes of AO calculations.
  */
 public sealed interface BakedNormals {
     BakedNormals UNSPECIFIED = new PerQuad(0);
 
-    int normals(int vertex);
+    /**
+     * @return The packed normal of the given vertex. See {@link #unpack}. It can also be {@linkplain #isUnspecified(int) unspecified}.
```

Comment:
Using only `@return` is discouraged because it means that the generated javadoc would get no one-sentence summary.